synths/default-synths.scd

```diff
@@ -75,3 +75,197 @@ SynthDef(\gabor, { |out, sustain = 0.03, note = 0, amp = 1, pan = 0, width = 0.3
 	)
 }).add;
 );
+
+(
+var numChannels = ~dirt.numChannels;
+
+// list of new SynthDefs, and in parentheses the special parameters they accept
+// ("!" means it's not a standard Tidal param and needs to be defined)
+// most SynthDefs use speed to scale the overall envelope, and many use accelerate to provide a pitch glide
+//
+// tutorial1, tutorial2, tutorial3, tutorial4, tutorial5(f!),
+// supermandolin(detune), supergong(voice,decay), superpiano(velocity,detune,muffle!,stereo!), superhex,
+// superkick(pitch1,decay), superhat, supersnare(decay), superclap(delay,pitch1), super808(voice), supersiren
+
+// a very basic starting point, just a 440Hz sine wave
+SynthDef(\tutorial1, {|out|
+	var sound = SinOsc.ar(440.0);
+	OffsetOut.ar(out, DirtPan.ar(sound, ~dirt.numChannels, 0.5));
+}).add;
+
+// now let's add some better duration control, and support for pan
+// Tidal's synth parameters magically show up as arguments to the SynthDef!
+// overall gain is handled elsewhere by SuperDirt, so we don't need it in the envelope
+SynthDef(\tutorial2, {|out, begin=0, end=1, pan, speed=1 |
+	var dur = (end-begin)/speed;
+	var env = EnvGen.ar(Env.linen(0.01, dur, 0.01, 1,-3), timeScale:dur, doneAction:2);
```

Comment:
OK, I didn't realize that.  I think that'll simplify the examples, so I'll change them.  I overload `speed` sometimes, but I hardly ever do anything with `begin` or `end`.